.github/workflows/tests.yml

```diff
@@ -11,15 +11,21 @@ on:
 
 jobs:
   linux:
-    name: Linux py${{ matrix.PYTHON_VERSION }} tests
-    runs-on: ubuntu-latest
+    name: ${{ matrix.PLATFORM }} py${{ matrix.PYTHON_VERSION }}
+    runs-on: ${{ matrix.PLATFORM }}
     env:
       CI: True
       PYTHON_VERSION: ${{ matrix.PYTHON_VERSION }}
     strategy:
       fail-fast: false
       matrix:
         PYTHON_VERSION: ['3.5', '3.6', '3.8']
```

Comment:
again, maybe and add 3.9, and drop 3.6?

Reply:
I tried and it did not work :-/

Reply:
Cool/bad! The utter fiasco that was tornado vs 3.8 makes me want to find these things out sooner rather than later. Maybe we can keep it, and just run it against linux, and expect failure? Gotta start somewhere!

Reply:
I mean I tried

https://github.com/actions/setup-python

```
steps:
- uses: actions/checkout@v2
- uses: actions/setup-python@v2
  with:
    python-version: '3.9.0-beta.4'
```

And the install of that python did not work.

Reply:
Maybe try `3.9.0-beta.5`? The [manifest](https://github.com/actions/python-versions/blob/main/versions-manifest.json#L3) is the Source of Truth.

Reply:
trying this out on #103...

Reply:
yeah, no dice. weird.